Firmware/temperature.cpp

```diff
@@ -782,15 +797,18 @@ void manage_heater()
         if(current_temperature_bed >= target_temperature_bed)
         {
           soft_pwm_bed = 0;
+		  timer02_set_pwm0(soft_pwm_bed << 1);
```

Comment:
nebylo by lépe (v tomto případě) použít:
timer02_set_pwm0(0);

Reply:
Nebylo, takle je vsude kde se nastavuje vystupni hodnota stejny kod a to bylo cilem, aby byla zmena prehledna. Ten kod jeste neni finalni, puvodni soft_pwm casem odstranime.